### PR TITLE
LPS-114831

### PR DESCRIPTION
**Pull request by petershin**
@gamerson

### CODE REVIEW

File: modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java [hunk 1]
```diff
@@ -546,8 +577,7 @@ public void handle(HttpExchange httpExchange) throws IOException {
 	}
 
 	private static URL _getHttpServerUrl(String contextPath) throws Exception {
-		return new URL(
-			"http", "localhost.localdomain", _HTTP_SERVER_PORT, contextPath);
+		return new URL("http", "127.0.0.1", _HTTP_SERVER_PORT, contextPath);
```
Comment (gamerson) [inline modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java:580]: Oops I left these in my commit.  I had to change this to get it to work on my machine. Do you want to change it back?

**Comment by petershin** [comment]
ci:stop

File: modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java (hunk 1 above)
Comment (petershin) [reply modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java:580]: Yes, please change it back and resend

Comment (gamerson) [reply modules/util/portal-tools-bundle-support/src/test/java/com/liferay/portal/tools/bundle/support/commands/BundleSupportCommandsTest.java:580]: done and sent. https://github.com/petershin/liferay-portal/pull/1025

**Pull request closed without merging**